app.bluebubbles.BlueBubbles.yml

```diff
@@ -0,0 +1,104 @@
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/objectbox/objectbox-c/releases/download/v0.16.0/objectbox-linux-x64.tar.gz
+        sha256: b597c10607bbba71b238c935581ee7bde5ea07211d899bd4581685e495da01f8
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://github.com/objectbox/objectbox-c/releases/download/v0.16.0/objectbox-linux-aarch64.tar.gz
+        sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+  - name: libidn
+    buildsystem: simple
+    build-commands:
+      - './configure --prefix=/app --disable-static && make && make install'
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/libidn/libidn-1.34.tar.gz
+        archive-type: tar-gzip
+        sha256: 3719e2975f2fb28605df3479c380af2cf4ab4e919e1506527e4c7670afff6e3c
+    cleanup:
+      - /bin/*
+      - /include/*
+      - /share/*
+  - name: vlc
+    buildsystem: simple
+    build-commands:
+      - 'BUILDCC="/usr/bin/gcc -std=gnu99" ./configure --prefix=/app --disable-a52 --disable-lua && make clean && make libcompat && make install'
+    sources:
+      - type: archive
+        url: http://download.videolan.org/pub/videolan/vlc/3.0.16/vlc-3.0.16.tar.xz
+        sha256: ffae35fc64f625c175571d2346bc5f6207be99762517f15423e74f18399410f6
+    cleanup:
+      - /bin/*
+      - /include/*
+      - /share/*
+      - /lib/vlc
+      - /lib/*.la
+  - name: bluebubbles
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p bluebubbles'
+      - 'tar -xf bluebubbles-linux-portable.tar.gz -C bluebubbles'
+      - 'cp -r bluebubbles /app/'
+      - 'chmod +x /app/bluebubbles/bluebubbles'
+      - 'mkdir -p /app/bin'
+      - 'ln -s /app/bluebubbles/bluebubbles /app/bin/bluebubbles'
+      - 'mkdir -p /app/share/metainfo'
+      - 'cp -r app.bluebubbles.BlueBubbles.metainfo.xml /app/share/metainfo/'
+      - 'mkdir -p /app/share/icons/hicolor/64x64/apps'
+      - 'cp -r 64x64.png /app/share/icons/hicolor/64x64/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/128x128/apps'
+      - 'cp -r 128x128.png /app/share/icons/hicolor/128x128/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/256x256/apps'
+      - 'cp -r 256x256.png /app/share/icons/hicolor/256x256/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/512x512/apps'
+      - 'cp -r 512x512.png /app/share/icons/hicolor/512x512/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/applications'
+      - 'cp -r app.bluebubbles.BlueBubbles.desktop /app/share/applications/'
+      - 'mkdir -p /app/share/appdata'
+      - 'cp -r app.bluebubbles.BlueBubbles.metainfo.xml /app/share/appdata/'
+    sources:
+      - type: file
+        url: https://github.com/BlueBubblesApp/bluebubbles-app/releases/download/v1.9.1%2B30/bluebubbles-linux-portable.tar.gz
```

Comment:
I have doubt how "portable" that is.

`/lib` contains `ELF 64-bit LSB shared object, x86-64,`

So I have doubt of the validaty of aarch64 support here.

It's sad, but it's flutter.